emcc.py

```diff
@@ -1910,6 +1913,8 @@ def compile_source_file(i, input_file):
         def supported(f):
           if f in SUPPORTED_LINKER_FLAGS:
             return True
+          if f.startswith('-l') or f.startswith('-L'):
+            return False
```

Comment:
I don't understand this change?

Reply:
One of that things this change does is allow the user's -l and -L flags to propogate through to linker flags.  However, for fastcomp we want to strip these out here as they are handled elsewhere.